python/eos/analysis_file_TEST.d/valid-analysis-file.yaml

```diff
@@ -129,8 +129,8 @@ predictions:
           q2_max: 25.0
 
 steps:
-  - title: 'Sample from CKM-all posterior'
-    id: 'CKM-all.sample'
+  - title: 'Sample from CKM-all and WET-all posteriors'
+    id: 'CKM-all,WET-all.sample'
```

Comment:
I'm not sure that I like this short hand. I used the dot as a separator for personal preference only. Not sure that everybody will.

Reply:
I'm open to different conventions. I guess the main place it matters is not here, but the example in the documentation, there we should use some sensible convention